### PR TITLE
Image hero  added to the home and about page

### PR DESCRIPTION
**Pull request by Monika39-maker**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -4,8 +4,9 @@ import About from "./pages/About";
 import Donations from "./Donations";
 import ThankYou from "./pages/ThankYou";
 import Home from "./pages/Home";
-import About from "./pages/About";
```
Comment (HThanki) [inline client/src/App.js]: Hi @Monika39-maker, if we delete line 7 the /about page will not render

**Review by HThanki — COMMENTED** [review]
Hi @Monika39-maker, if we delete line 7 the /about page will not render

File: client/src/App.js (hunk 1 above)
Comment (Monika39-maker) [reply client/src/App.js]: so, leave it there,pls

Comment (Monika39-maker) [reply client/src/App.js]: I might have mistakenly deleted it when solving merge request

**Pull request merged**